自定义两个新的异常FileError和NetworkError
